back/config/routes.rb

```diff
@@ -28,6 +28,7 @@
         resources :permissions, param: :permission_action do
           get 'requirements', on: :member
           get 'schema', on: :member
+          get 'custom_fields', on: :member
```

Comment:
I needed to create new endpoints, replicating the functionality of the existing `/schema` ones for the different scopes (global, project, etc.)